README.md

```
@@ -199,6 +216,17 @@ Services will be available at:
 - Firecrawl API: `http://localhost:50102`
 - MCP Server: `http://localhost:50107`
 - Webhook Bridge: `http://localhost:50108`
+- Web UI: `http://localhost:3000`
+
+### Accessing the Web UI
+
+The web interface runs at http://localhost:3000 (configurable via `WEB_PORT`).
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)